tests/manual/web/WebSocketTestApp.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file is new in 2024, so you can remove the `2022,`

Reply:
done